dart_ping/lib/src/models/ping_parser.dart

```diff
@@ -121,6 +130,15 @@ class PingParser {
             );
           }
 
+          // Host or Network Unreachable
+          if (data.contains(unreachableStr)) {
```

Comment:
These if statement should probably become if/else unless we can guarantee that the presence of one truthy contains() negates the others. Reason being that we don't want one line of ping output to result in two separate PingData events.